doc/plotting.rst

```diff
@@ -580,6 +602,67 @@ they have been plotted.
 TODO: add an example of using the ``map`` method to plot dataset variables
 (e.g., with ``plt.quiver``).
 
+.. _plot-dataset:
+
+Datasets
+--------
+
+``xarray`` has limited support for plotting Dataset variables against each other.
+Consider this dataset
```

Comment:
How about shortening this whole code block to `xr.tutorial.open_dataset(...)`? would make it more readable I think

Reply:
So make an additional dataset to upload to `xarray-data`? None of the existing ones have multiple variables.

Reply:
Maybe finish up this wind dataset from ERA5? https://github.com/pydata/xarray-data/pull/8

Reply:
Dunno. Seems like downloading a 40MB file is an overkill. I'd simply put the code for generating the  current dataset on the fly in xr.tutorial